env_vars.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# set these variables before running the script
+LOCAL_BASKERVILLE_PATH="/home/jlinder/baskerville"
+LOCAL_USER="jlinder"
+
+# create env_vars sh scripts in local conda env
+mkdir -p "$CONDA_PREFIX/etc/conda/activate.d"
+mkdir -p "$CONDA_PREFIX/etc/conda/deactivate.d"
+
+file_vars_act="$CONDA_PREFIX/etc/conda/activate.d/env_vars.sh"
+if ! [ -e $file_vars_act ]; then
+    echo '#!/bin/sh' > $file_vars_act
+fi
+
+file_vars_deact="$CONDA_PREFIX/etc/conda/deactivate.d/env_vars.sh"
+if ! [ -e $file_vars_deact ]; then
+    echo '#!/bin/sh' > $file_vars_deact
+fi
+
+# append env variable exports to /activate.d/env_vars.sh
+echo "export BASKERVILLE_DIR=$LOCAL_BASKERVILLE_PATH" >> $file_vars_act
+echo 'export PATH=$BASKERVILLE_DIR/src/baskerville/scripts:$PATH' >> $file_vars_act
+echo 'export PYTHONPATH=$BASKERVILLE_DIR/src/baskerville/scripts:$PYTHONPATH' >> $file_vars_act
+
+echo "export BASKERVILLE_CONDA=/home/$LOCAL_USER/anaconda3/etc/profile.d/conda.sh" >> $file_vars_act
```

Comment:
Maybe just ask for this path directly and let go the LOCAL_USER variable. I don't know how other people's systems will work for this.

Reply:
Will do!